### PR TITLE
Add docs for islamic_prayer_times sensor platform

### PR DESCRIPTION
**Pull request by uchagani**
**Description:**
Adding documentation for new Islamic prayer times sensor platform

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19444

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.islamic_prayer_times.markdown [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      description: Show the maghrib prayer time for today.
+    isha:
+      description: Show the isha prayer time for today.
+    midnight:
+      description: Show the midnight for today which is the end of isha prayer.  This is a calculated field and is not the same as 12:00AM.
+calculation_method:
+  required: false
+  default: 'isna'
+  type: string
+  description: The calculation method used for prayer times.  Must be one of: 'karachi', 'isna', 'mwl', 'makkah'.
```
Comment (ludeeus) [inline source/_components/sensor.islamic_prayer_times.markdown]:
```suggestion
  description: "The calculation method used for prayer times.  Must be one of: 'karachi', 'isna', 'mwl', 'makkah'."
```
The stray `:` here will give a netlify error if not quoted

File: source/_components/sensor.islamic_prayer_times.markdown [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+sharing: true
+footer: true
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+ha_release: "0.85"
+---
+
+The Islamic Prayer Times (`islamic_prayer_times`) sensor platform displays the various prayer times for Muslims as sensors.
+
+This component calculates prayer times using the following calculation methods:
```
Comment (MartinHjelmare) [inline source/_components/sensor.islamic_prayer_times.markdown]: It's a platform.

Comment (uchagani) [reply source/_components/sensor.islamic_prayer_times.markdown]: fixed

**Review by MartinHjelmare — APPROVED** [review]
Great!

**Comment by MartinHjelmare** [comment]
Can be merged when build passes.

**Comment by uchagani** [comment]
@MartinHjelmare build has passed.

**Pull request merged**